pact/Pact/Core/Syntax/ParseTree.hs

```diff
@@ -195,9 +195,8 @@ data PactStep i
 
 data DefPact i
   = DefPact
-  { _dpName :: Text
-  , _dpArgs :: [MArg]
-  , _dpRetType :: Maybe Type
+  { _dpSpec :: MArg i
```

Comment:
Haddocs would be nice here, since it's not obvious from the name `_dbSpec` nor the type `MArg` that this field describes the DefPact's name.